common/source/docs/common-camera-runcam.rst

```diff
@@ -0,0 +1,64 @@
+
+Not all features of the protocol are currently supported in ArduPilot and not all features of the protocol are supported on all cameras. In particular it is important to distinguish between cameras that support the "5-button protocol" (generally camera -only devices such as the RunCam Racer) and cameras that support the "2-button protocol" (generally split-style cameras). These two protocols originated from the 5-key external OSD keyboard and 2 buttons present on split-style cameras. Equally important is to understand that the RunCam device protocol is almost a direct mapping from the hardware - so the way that you would physically press a button is the way that the protocol is invoked. The only caveat here is that the 5-button protocol recognizes keys being released, whereas the 2-button protocol does not, leading to some limitations in the way that menus and options can be navigated.
+
+Lately similar to TBS Smartaudio and IRC Tramp other companies (for instance Caddx) have started to adopt support for the protocol in their cameras.
+Currently supported features in ArduPilot
+
+* Full camera OSD control
+* Starting/Stopping recording on split style cameras
+
+Setup
+=====
+
+* One free non-inverted UART on the FC is required (TX and RX). Connect TX to RX, RX to TX and GND to GND. Pinouts for serial ports on The Cube can be found `here <https://ardupilot.org/copter/docs/common-pixhawk-serial-names.html>`__.
+
+.. image:: ../../../images/camera-runcam-fc-setup.jpg
+    :target:  ../_images/camera-runcam-fc-setup.jpg
+    :width: 450px
+
+  On some camera-only devices the pinout is labelled such that TX should be connected to TX and RX to RX. If your RunCam device is not recognized try reversing the signal wires. Not all cameras support a UART ground pin - they expect to get ground from the power supply.
+  The camera must be in UART control mode. On split-style cameras this should be the default. On camera-only devices you will need to follow the procedure for putting the device in UART control mode. Consult your RunCam manual for more details.
+  Note that so-called "Joystick control" is not supported and some devices only support joystick control, so be careful when selecting the RunCam model you are going to use. The following devices have been tested successfully - RunCam Split 3s Micro, RunCam Split 2s, RunCam Racer 3.
+
+* In order to recognize the RunCam serial protocol on the UART, set the UART serial protocol to 26. So for instance for serial port 5, set :ref:`SERIAL5_PROTOCOL <SERIAL5_PROTOCOL>` 26 (= RunCam protocol). Adjust the serial port's protocol parameter to 26, for the serial port that your RunCam's serial wires are connected to, as required. The correlation between serial port numbering and UART physical ports for your autopilot should be documented in its description page linked :ref:`here <common-autopilots>`.
```

Comment:
move =  out of parenthetical and in front of 26...
Sentence beginning with Adjust is redundant...

Reply:
Fixed